### ISSUE TITLE
LFR: LFR: Light's Breach

### ISSUE THREAD

**Issue by Trid3nt1** [SavedInstances/SavedInstances#81]
Currently showing 2 LFR: inside the tables

https://i.imgur.com/w1moVtY.png

**Comment by AlexAtTheBux** [comment]
This is puzzling me, as not only does it have two "LFR: " in the title, but it should also be "LFR: Antorus, the Burning Throne".

Anyone able to figure out what is wrong here?

**Comment by Razariel** [comment]
the parentID for Antorus is not 1610, that is the ID of Light's Breach, which is why it looks like this

using the same method ceylina used to get the ID of ToS here https://github.com/SavedInstances/SavedInstances/issues/13, the ID should be 1642, which in fact works ingame http://oi64.tinypic.com/2cr2bk7.jpg

**Comment by Trid3nt1** [comment]
#82 
I've submitted the pull request and added the commands to the comments for easier future use. Nicely figured out @Razariel

**Comment by Trid3nt1** [comment]
What's interesting, for future reference. The LFR Parent ID is the same as the Mythic Parent ID.
1640, the original Parent ID before being changed to 1610 - Referred to Antorus Normal.

Index;
40. Light's Breach (1610) 
41. Forbidden Descent (1611) 
42. Hope's End (1612) 
43. Seat of the Pantheon (1613)

Parent ID's;
Normal: 1640 Antorus
Heroic: 1641 Antorus
Mythic: 1642 Antorus

To confirm the change to the parent ID is correct this time round, you can compare to a previous LFR.

  [1290] = { total=3, base=1,  parent=1353, altid=nil }, -- NH1: Arcing Aqueducts
  [1291] = { total=3, base=4,  parent=1353, altid=nil, remap={ 1, 2, 3 } }, -- NH2: Royal Athenaeum
  [1292] = { total=3, base=7,  parent=1353, altid=nil, remap={ 1, 2, 3 } }, -- NH3: Nightspire
  [1293] = { total=1, base=10, parent=1353, altid=nil, remap={ 1 } }, -- NH4: Betrayer's Rise

Has the same Parent ID as:
Mythic: 1353 The Nighthold

So, TL;DR for those wanting to help in future - Mythic / LFR Parent IDs are the same. I personally didn't know this either, so if nothing else - TIL.

**Comment by Stanzilla** [comment]
Well done everyone!
